selfservice/strategy/oidc/provider.go

```diff
@@ -52,6 +57,8 @@ type Claims struct {
 	UpdatedAt           int64                  `json:"updated_at,omitempty"`
 	HD                  string                 `json:"hd,omitempty"`
 	Team                string                 `json:"team,omitempty"`
+	Nonce               string                 `json:"nonce,omitempty"`
+	NonceSupported      bool                   `json:"nonce_supported,omitempty"`
```

Comment:
Is this an actual claim returned by Apple/Google?

Reply:
I see, this is returned by Apple. I don't think this is a standard claim but rather something specific to Apple. In that case, I would suggest to only handle this claim in the apple provider. The problem here is that this bool value will default to "false", which disables the nonce check if the value is not available (which is the case with any other platform). So basically, this is a bypass for the nonce check, built in and enabled per default.